src/Symfony/Component/Messenger/Transport/AmqpExt/AmqpReceiver.php

```diff
@@ -74,7 +81,11 @@ public function get(): iterable
     public function ack(Envelope $envelope): void
     {
         try {
-            $this->connection->ack($this->findAmqpEnvelope($envelope));
+            /* @var AmqpReceivedStamp $amqpReceivedStamp */
+            $this->connection->ack(
+                $this->findAmqpEnvelope($envelope, $amqpReceivedStamp),
+                $amqpReceivedStamp->getQueueName()
```

Comment:
This tripped me up :). Minor, but I'd prefer the method to now be `findAmqpReceivedStamp()` and then to use that here to get the AmqpEnvelop & queue name

Reply:
Indeed. Made it much simpler.